cassarrow/impl.py

```diff
@@ -58,7 +58,7 @@ def get_arrow_type(dtype: CassandraTypeType) -> pa.DataType:
         raise TypeError(f"{typename}: {dtype}")
 
 
-def column_metadata_to_schema(column_metadata: list[tuple]) -> pa.Schema:
+def column_metadata_to_schema(column_metadata: list) -> pa.Schema:
```

Comment:
I'd rather we keep the type hints explicit. With python 3.8 you can use something like:
```
from typing import List, Tuple


def column_metadata_to_schema(column_metadata: List[tuple]):
   ...

def metadata_to_schema(names: List[str], dtypes: List[CassandraTypeType]):
   ...
```

And same with `tuple[pa.RecordBatch] to Tuple[pa.RecordBatch]`

This is compatible with python 3.8+

Reply:
Makes sense! I made the updates in the last commit, same branch patch-1